### PR TITLE
Update README file.

### PR DESCRIPTION
**Pull request by rcorral**
Added helpful links and a gif so that the community can quickly see what it's like to use the UI.

[See updated readme](https://github.com/dcos/dcos-ui/blob/rafa/update-readme/README.md)

### CODE REVIEW

**Comment by jfurrow** [comment]
Nice and clean, I like the update!

**Comment by jfurrow** [comment]
run integration tests

**Comment by rcorral** [comment]
um.. i just realized I used an EE cluster for the gif...dammit...and I tried so hard to get the timing right on that one.

**Comment by rcorral** [comment]
Fixed up gif!

File: README.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
 # DC/OS UI [![Velocity](http://jenkins.mesosphere.com/buildStatus/icon?job=public-dcos-ui-master)](http://jenkins.mesosphere.com/view/DCOS%20UI/job/public-dcos-ui-master/)
 
-## Requirements
+The User Interface for The Datacenter Operating System.
 
-Node 4.4.x and NPM 3.9.x (and above) are **required**. We suggest using [nvm](https://github.com/creationix/nvm) or [n](https://github.com/tj/n) to keep multiple Node versions on your system.
+The DC/OS UI is used by datacenter operators to administer and manage a datacenter. Developers use DC/OS UI to manage, deploy and debug their applications. The DC/OS UI sits at the top of the [DC/OS stack](https://dcos.io/docs/1.8/overview/architecture/) along with the [CLI](github.com/dcos/dcos-cli).
+
+* [Issue tracker](https://dcosjira.atlassian.net). Use component `dcos-ui`.
```
Comment (nLight) [inline README.md]: should it be now https://jira.dcos.io/ ?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+* [Documentation](https://dcos.io/docs/1.8/usage/webinterface/)
```
Comment (nLight) [inline README.md]: I'd point master to 1.9

File: README.md [hunk 3]
```diff
@@ -1,12 +1,22 @@
 # DC/OS UI [![Velocity](http://jenkins.mesosphere.com/buildStatus/icon?job=public-dcos-ui-master)](http://jenkins.mesosphere.com/view/DCOS%20UI/job/public-dcos-ui-master/)
 
-## Requirements
+The User Interface for The Datacenter Operating System.
 
-Node 4.4.x and NPM 3.9.x (and above) are **required**. We suggest using [nvm](https://github.com/creationix/nvm) or [n](https://github.com/tj/n) to keep multiple Node versions on your system.
+The DC/OS UI is used by datacenter operators to administer and manage a datacenter. Developers use DC/OS UI to manage, deploy and debug their applications. The DC/OS UI sits at the top of the [DC/OS stack](https://dcos.io/docs/1.8/overview/architecture/) along with the [CLI](github.com/dcos/dcos-cli).
```
Comment (nLight) [inline README.md]: https://dcos.io/docs/1.9/overview/architecture/ ?

File: README.md [hunk 4]
```diff
@@ -35,89 +45,24 @@ Node 4.4.x and NPM 3.9.x (and above) are **required**. We suggest using [nvm](ht
...
-          ]
-        }
-      ]
-    }
-    ```
-
+## Contributing
 
-## Testing
+[Contributing Guidelines](./.github/CONTRIBUTING.md)
```
Comment (nLight) [inline README.md]: The link doesn't work, but that's different PR right?

**Comment by MatApple** [comment]
Added hold merge till comments are addressed 👍

**Comment by mlunoe** [comment]
Also, I see some fixes that have merged, but looks weird in the gif. Could we update it with latest master?

File: README.md (hunk 4 above)
Comment (orlandohohmeier) [inline README.md]: The file path is wrong.

Comment (rcorral) [inline README.md]: Done!

File: README.md (hunk 2 above)
Comment (rcorral) [inline README.md]: Done!

File: README.md (hunk 1 above)
Comment (rcorral) [inline README.md]: Done!

File: README.md (hunk 3 above)
Comment (rcorral) [inline README.md]: Done!

**Comment by rcorral** [comment]
Addressed all comments and updated gif.

**Comment by Poltergeist** [comment]
Added label since the other label is 2 month old I removed it, we should get a second opinion from today.

**Pull request merged**